### PR TITLE
Add root .editorconfig file.

### PR DESCRIPTION
**Pull request by amio**
[EditorConfig](http://EditorConfig.org) should do some help to restrict coding style :)

### CODE REVIEW

**Comment by bobthecow** [comment]
I'd just make everything two-space indent, tbh. We can put in exceptions if we ever need 'em :)

**Comment by bucaran** [comment]
@bobthecow Agree.

@amio 
Isn't there a way to combine
```
[*.fish]
indent_style = space
indent_size = 2


# 2-space for .load file
[*.load]
indent_style = space
indent_size = 2
```
?

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# 2-space for .fish file
+[*.fish]
```
Comment (bpinto) [inline .editorconfig]: Does `[*.fish, *.load]` works?

**Comment by bobthecow** [comment]
```ini
# EditorConfig is awesome: http://EditorConfig.org

# top-most EditorConfig file
root = true

# Two-space indent, Unix-style newlines, and a newline ending every file
[*]
end_of_line = lf
insert_final_newline = true
indent_style = space
indent_size = 2
```

File: .editorconfig (hunk 1 above)
Comment (amio) [inline .editorconfig]: `[ *.{fish,load} ]` will work

Comment (bucaran) [inline .editorconfig]: :+1: Nice

**Comment by amio** [comment]
@bobthecow I thought declare them separately could make rules clearly, and since I didn't know if we all prefer to use 2-space indent in most cases. Well, I know now :)

Force updated~

**Comment by bpinto** [comment]
Thanks @amio! This is great.

**Pull request merged**